python_homeworks/homework_4/task_1.py

```diff
@@ -0,0 +1,10 @@
+# 1) Вычислить число c заданной точностью d
+# Пример:
+# при d = 0.001, π = 3.142 10^(-1) ≤ d ≤10^(-10)
+
+from math import pi
+
+a = str(input('Введите число: '))
```

Comment:
```suggestion
a = input('Введите число: ')
```
Функция ввода данных всегда возвращает объект типа `str`.